charts/spire/values.yaml

```diff
@@ -19,17 +19,38 @@ spire-server:
   controllerManager:
     enabled: true
 
-spire-agent:
+agent-downstream:
```

Comment:
I prefer not to rename this.

Reply:
so, there would be agent and agent-upstream?